### DISCUSSION TITLE
How to use ProtoZOA?

### DISCUSSION THREAD

**Discussion by LJAndy** [hathach/tinyusb#1549] [General]
I got the ProtoZOA,  how can i check the messages with windows? need install some drives?  I can find the device, but i can not find any MIDI port.

**Comment by pigrew** [comment]
I don't think any of us know about the protoZOA, and I can't find any detailed design documentation on it... You should contact the ProtoZOA designer/manufacturer.

https://amenote.com/?page_id=74

(Also, personally, I'm not sure which versions of Windows have MIDI 2.0 drivers, or even what software would support MIDI 2.0)

**Reply by LJAndy** [reply]
Thanks. Do you know mac OS support MIDI2.0?

**Reply by pigrew** [reply]
I don't know, but it seems not. There seems to be some custom drivers that come with digital audio workstation software?

Based on the ProtoZoa docs, the device should be backwards compatible with the MIDI 1.0 USB class, so you should be able to do something...

**Comment by senorblasto** [comment]
> I got the ProtoZOA, how can i check the messages with windows? need install some drives? I can find the device, but i can not find any MIDI port.

Share the source code and I'm sure people will be happy to work with you to get it working
